.github/workflows/coverity-scan.yml

```
@@ -0,0 +1,63 @@
+name: coverity-scan
+
+# Controls when the action will run. 
+on:
+  # Run this action on a schedule (we're allowed a maximum of two per day)
+  #schedule:
+  #  - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
```

to avoid this running (and failing) on clones:
```suggestion
    if: github.repository == 'MapServer/MapServer'
    steps:
```

---

Wouldn't we want clones - those with the token - be able to run this?

---

ah, perhaps there's a way to write a condition on the availability of the ${{ secrets.COVERITY_SCAN_TOKEN }} . Like done in https://github.com/entropia/tip-toi-reveng/commit/2582843896dbbb34581edf2335e3a658d6168747

---

will take a look...